lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,7 +18,16 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
-
+    if not isinstance (text, str):
+        return None
+    tokenized_text = []
+    punctuation_digits = '''!()-[]{};:'",<>./\?@#$%^&*_~1234567890=+'''
```

Comment:
It is not alphas, right? What if you can use sum function to identify that a symbol is alpha